lms/templates/problem.html

```diff
@@ -30,6 +30,36 @@ <h3 class="hd hd-3 problem-header" id="${ short_id }-problem-title" aria-describ
       <button type="button" class="submit btn-brand" data-submitting="${ submit_button_submitting }" data-value="${ submit_button }" data-should-enable-submit-button="${ should_enable_submit_button }" aria-describedby="submission_feedback_${short_id}" ${'' if should_enable_submit_button else 'disabled'}>
           <span class="submit-label">${ submit_button }</span>
       </button>
+
+      % if submit_disabled_cta:
+        % if 'form_values' in submit_disabled_cta:
+          <form class="submit-cta" method="post" action="${submit_disabled_cta['link']}">
+            <input type="hidden" id="csrf_token" name="csrfmiddlewaretoken" value="${csrf_token}">
+            % for form_name, form_value in submit_disabled_cta['form_values'].items():
+                <input type="hidden" name="${form_name}" value="${form_value}">
+            % endfor
+            <button class="submit-cta-link-button btn-brand">${submit_disabled_cta['link_name']}
+              <span class="submit-cta-description" tabindex="0" role="note" aria-label="description">
+                <span data-tooltip="${submit_disabled_cta['description']}" data-tooltip-show-on-click="true"
+                    class="fa fa-info-circle fa-lg" aria-hidden="true">
+                </span>
+              </span>
+              <span class="sr">(${submit_disabled_cta['description']})</span>
+            </button>
+          </form>
+        % else:
+          <div class="submit-cta">
+            <a href="${submit_disabled_cta['link']}">${submit_disabled_cta['link_name']}
+              <span class="submit-cta-description" tabindex="0" role="note" aria-label="description">
+                <span data-tooltip="${submit_disabled_cta['description']}" data-tooltip-show-on-click="true"
+                    class="fa fa-info-circle fa-lg" aria-hidden="true">
+                </span>
+                <span class="sr">(${submit_disabled_cta['description']})</span>
+              </span>
+            </a>
+          </div>
```

Comment:
It's not clear to me that a lack of `form_values` should imply a GET request rather than a POST request that just doesn't have any parameters.

At the least, our two template fragments should agree about it. If it does imply a GET, we should probably document that in the plugin docstring.

Reply:
Hmm. Yeah. I'm going to just collapse it to always be a POST, for now, but document that in the future we might want to add a separate `method` parameter in the plugin method.